src/main/java/uk/gov/hmcts/reform/unspec/service/flowstate/MainFlowState.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.unspec.service.flowstate;
+
+public enum MainFlowState {
```

Comment:
Maybe create a class to wrap all FlowStates and call it `FlowState`, then the enum inside can be called `Main`, and any other future subflow we will have?